doc/contributor/svm.md

```diff
@@ -86,5 +81,34 @@ Hello
 0.25 66824
 ```
 
-(The first number `real` is the number of actual seconds which have elapsed while the command
-runs, and the second `maximum resident set size` is the maximum amount of memory occupied while the command runs)
+(The first number `real` is the number of actual seconds which have elapsed
+while the command runs, and the second `maximum resident set size` is the
+maximum amount of memory occupied while the command runs.)
+
+There is no need to do so, but you can actually also compile your own copy of
+the SVM version of TruffleRuby using a tool distributed as part of GraalVM and
+the Java version of TruffleRuby from GraalVM.
+
+```
+$ native-image -H:Name=native-ruby --Language:ruby
+```
+
+`native-ruby` is the output file name.
+
+You can build a native image using SVM from a source distribution using:
+
+```
+$ jt build native
+```
+
+The disadvantages of the SVM version of TruffleRuby are:
+
+* It has lower peak performance, as the GC is simpler and some optimisations
+  such as compressed ordinary object pointers (OOPS) are not available.
+* You can't use Java interopability.
+* You can't use standard Java tools like VisualVM.
```

Comment:
Maybe worth adding sth about native tools can be used, like valgrind, etc. cc @nirvdrum